{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "           ConnectionResetError: [WinError 10054] Se ha forzado la interrupci\u00f3n de una conexi\u00f3n existente por el host remoto\n           urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'Se ha forzado la interrupci\u00f3n de una conexi\u00f3n existente por el host remoto', None, 10054, None))\n", "test_name": "test_base[404477902-1qazxsw2.]", "rerun": "0"}}, "suite_name": "tests/test_scheduling.py"}}}, "date": "May 29, 2023", "start_time": 1685377969.49131, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}